content/ytt/docs/latest/file-marks.md

```diff
@@ -65,15 +65,19 @@ Change type of file, affecting how `ytt` processes it.
 --file-mark '<path>:type=<file-type>'
 ```
 
-By default `file-type` is determined based on file extension. 
+By default `file-type` is determined based on file extension (and, as of v0.32.0, content).
 
 `file-type` can be: 
 - `yaml-template` (ext: `.yml`) — parsed as a YAML document containing `ytt` templating.
-- `yaml-plain` — parsed as a simple YAML document (not processed for templating).
+- `yaml-plain` (ext: `.yml`)  — parsed as a simple YAML document (not processed for templating).
 - `text-template` (ext: `.txt`) — parsed as a text document containing text templating.
-- `text-plain` — plain text (included in output, as is)
+- `text-plain` (ext: `.txt`) — plain text (included in output, as is)
 - `starlark` (ext: `.star`) — a Starlark source file (executed)
 - `data` — a text file that can be loaded by [`data.read()`](lang-ref-ytt.md#data)
+
+As of v0.32.0, `ytt` attempts to automatically detect when a file is a template:
```

Comment:
Can we make this definitive? Does ytt take automatically detect when a file is a template?

Are there exceptions that we need to point out?

Reply:
That's a great point. There's no reason to leave this ambiguous. If there's any `ytt` annotation, it's considered a template, otherwise plain YAML... there, that wasn't hard to say! :)

Will make that update. Thank you.